compiler/src/main/java/com/squareup/anvil/compiler/codegen/dagger/MembersInjectorGenerator.kt

```diff
@@ -92,6 +103,15 @@ internal class MembersInjectorGenerator : PrivateCodeGenerator() {
 
     val memberInjectorClass = ClassName(packageName, className)
 
+    // Lazily evaluated, only computed if any properties have any non-inject annotations
+    val propertyDescriptors by lazy {
+      clazz.requireClassDescriptor(module)
```

Comment:
I hoped to avoid resolving the class descriptor, but writing a function and copying all annotation arguments is tedious.

Reply:
Yeah I ran into the same and that's why I made this lazy. We could revisit if it's a major problem for anyone in the future.

Reply:
Yes, the lazy option is a good compromise in my opinion.